tools/install_deps/pytest.txt

```diff
@@ -3,4 +3,5 @@ pytest-xdist~=1.31
 scikit-learn~=0.22
 scikit-image~=0.15.0
 Pillow~=7.0.0
+opencv-python~=4.1.1
```

Comment:
It seems opencv is not used in your pull request, you can remove it.

Reply:
Removed!!!